contrib/devtools/symbol-check.py

```diff
@@ -73,6 +73,8 @@
 'ld-linux-riscv64-lp64d.so.1', # 64-bit RISC-V dynamic linker
 # bitcoin-qt only
 'libxcb.so.1', # part of X11
+'libxkbcommon.so.0', # keyboard keymapping
+'libxkbcommon-x11.so.0', # keyboard keymapping
```

Comment:
Could this diff be moved from the _"build: qt 5.12.10"_ commit to the previous _"build: Add xkbcommon 0.8.4"_ one?